applications/portfolioDrafts/portfolio/getPortfolioStep.ts

```diff
@@ -0,0 +1,54 @@
+import { GetCommand } from "@aws-sdk/lib-dynamodb";
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from "aws-lambda";
+import { ErrorCodes } from "../models/Error";
+import { PortfolioStep } from "../models/PortfolioStep";
+import { dynamodbDocumentClient as client } from "../utils/dynamodb";
+import { ApiSuccessResponse, ErrorResponse, ErrorStatusCode, SuccessStatusCode } from "../utils/response";
+
+const TABLE_NAME = process.env.ATAT_TABLE_NAME;
+const NO_SUCH_PORTFOLIO_STEP = new ErrorResponse(
+  { code: ErrorCodes.INVALID_INPUT, message: "Portfolio Step not found for this Portfolio Draft" },
+  ErrorStatusCode.NOT_FOUND
+);
+const NO_SUCH_PORTFOLIO = new ErrorResponse(
```

Comment:
I want to be careful about the Portfolio/PortfolioDraft language.  Can we call this...
```suggestion
const NO_SUCH_PORTFOLIO_DRAFT = new ErrorResponse(
```